src/components/Settings/Export/Tabs/BibtexTabPanel.tsx

```diff
@@ -0,0 +1,246 @@
+import { JournalFormatSelect } from '@components/CitationExporter';
+import { JournalFormatMap } from '@components/Settings/model';
+import { UserDataSetterEvent } from '@pages/user/settings/export';
+import { useStore } from '@store';
+import { Dispatch, useMemo, useState } from 'react';
+import { journalNameHandlingDescription, bibtexExportFormatDescription, maxAuthorDescription } from '../Description';
+import { KeyFormatInputApply } from '../KeyFormatInputApply';
+import { MaxAuthorCutoffSlider } from '../MaxAuthorCutoffSlider';
+
+export type IBibtexTabPanelProps = {
+  sampleBib: IDocsEntity['bibcode'];
+  dispatch: Dispatch<UserDataSetterEvent>;
+};
+
+export const BibtexTabPanel = ({ sampleBib, dispatch }: IBibtexTabPanelProps) => {
+  const {
+    bibtexJournalFormat,
+    bibtexKeyFormat,
+    bibtexAuthorCutoff: bibtexAuthorCutoffStr,
+    bibtexMaxAuthors: bibtexMaxAuthorsStr,
+    bibtexABSKeyFormat,
+    bibtexABSAuthorCutoff: bibtexABSAuthorCutoffStr,
+    bibtexABSMaxAuthors: bibtexABSMaxAuthorsStr,
+  } = useStore((state) => state.settings.user);
+
+  const bibtexAuthorCutoff = useMemo(() => parseInt(bibtexAuthorCutoffStr), [bibtexAuthorCutoffStr]);
+  const bibtexMaxAuthors = useMemo(() => parseInt(bibtexMaxAuthorsStr), [bibtexMaxAuthorsStr]);
+  const bibtexABSAuthorCutoff = useMemo(() => parseInt(bibtexABSAuthorCutoffStr), [bibtexABSAuthorCutoffStr]);
+  const bibtexABSMaxAuthors = useMemo(() => parseInt(bibtexABSMaxAuthorsStr), [bibtexABSMaxAuthorsStr]);
+
+  // If bibtex settings and bibtex Abs settings are the same
+  // we can use basic view
+  const canUseBasicMode = useMemo(
+    () =>
+      bibtexKeyFormat === bibtexABSKeyFormat &&
+      bibtexAuthorCutoff === bibtexABSAuthorCutoff &&
+      bibtexMaxAuthors === bibtexABSMaxAuthors &&
+      bibtexAuthorCutoff === bibtexMaxAuthors, // basic settings cutoff and max authors are the same
+    [],
+  );
+
+  const [isBasicMode, setIsBasicMode] = useState(canUseBasicMode);
+
+  const tabs = ['bibtex', 'bibtex abs'];
+  const [selectedTab, setSelectedTab] = useState<string>(tabs[0]);
+
+  // these params are used to query sample citation
+  const { journalFormat, keyFormat, authorcutoff, maxauthor } = useMemo(() => {
+    const journalFormat = JournalFormatMap[bibtexJournalFormat];
+    const keyFormat = isBasicMode || (!isBasicMode && selectedTab === 'bibtex') ? bibtexKeyFormat : bibtexABSKeyFormat;
+    const authorcutoff =
+      isBasicMode || (!isBasicMode && selectedTab === 'bibtex') ? bibtexAuthorCutoff : bibtexABSAuthorCutoff;
+    const maxauthor =
+      isBasicMode || (!isBasicMode && selectedTab === 'bibtex') ? bibtexMaxAuthors : bibtexABSMaxAuthors;
+    return { journalFormat, keyFormat, authorcutoff, maxauthor };
+  }, [
+    bibtexJournalFormat,
+    bibtexKeyFormat,
+    bibtexAuthorCutoff,
+    bibtexMaxAuthors,
+    bibtexABSKeyFormat,
+    bibtexABSAuthorCutoff,
+    bibtexABSMaxAuthors,
+    selectedTab,
+  ]);
```

Comment:
Same here, I'd remove the memo since this is all basic logic, you can roll it into a helper file or something if you want to clean up the component a bit, but this should be fine to leave here if you'd like